app/views/kaui/layouts/kaui_header.html.erb

```diff
@@ -8,6 +8,8 @@
   <%= javascript_include_tag 'application' %>
   <%= csrf_meta_tags %>
 
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/flexibility/1.0.6/flexibility.js"></script>
```

Comment:
Is there a packaged version we could bundle? When Kaui is deployed to private datacenters, connectivity to the outside is most likely restricted.